nasa-jpl/explorer-1
# Fix/workaround for nuxt module

This PR addresses issues with the nuxt module when publishing to npm.

The solution is a workaround:

- The nuxt module is still not operational
- I did set up exports for `@explorer-1/vue` in [this commit](https://github.com/nasa-jpl/explorer-1/commit/dfbae91d5314dedea616b6af5a2fe0902cd66674). Calling this out in case we want to revisit doing this in the future.
- I decided to revert the export changes to the `@explorer-1/vue` package and instead use it directly in `@www/frontend` in a local module. This was the path of least resistance that would allow us to continue working in a dev env.

### To test

1. `pnpm i` 
2. `make vue-storybook` to make sure storybook and components still work
